src/test/end-to-end/codecept.conf.js

```diff
@@ -14,6 +14,33 @@ exports.config = {
       ignoreHTTPSErrors: true,
       windowSize: '1920x1080',
     },
+    /*Puppeteer: {
```

Comment:
I assumed I removed the puppeteer config, could we remove this instead of commenting out?